code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -72,13 +76,35 @@
 	if(A.totalStageSpeed() >= 6)
 		power = 2
 
+/datum/symptom/heal/starlight/proc/CanTileHeal(turf/T)
+	var/area/area_to_check = get_area(T)
+	if(isspaceturf(T) || area_to_check.outdoors) // Outdoors covers lavaland and unroofed areas but with tiles under, while space covers normal space and those caused by explosions
```

Comment:
I'm not sure I want to give this to lavaland, since my gut says miners will just abuse it. If this is a more general catchall I'm more ok with it

Reply:
The original intent for the outdoors check was as a catchall to catch unroofed areas in space, and then I checked and found out that lavaland also had that